tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -122,3 +122,46 @@ Feature: spaces management
       | David | viewer  |
       | Edith | viewer  |
     And "Alice" logs out
+
+
+  Scenario: admin user can manage the spaces created by other space admin user
+    Given "Admin" creates following users using API
+      | id    |
+      | Alice |
+      | Brian |
+      | Carol |
+    And "Admin" assigns following roles to the users using API
+      | id    | role        |
+      | Alice | Admin       |
+      | Brian | Space Admin |
+      | Carol | Space Admin |
+    When "Brian" logs in
+    And "Brian" creates the following project spaces using API
+      | name   | id     |
+      | team A | team.a |
+    When "Carol" logs in
+    And "Carol" creates the following project spaces using API
+      | name   | id     |
+      | team B | team.b |
+    When "Alice" logs in
+    And "Alice" opens the "admin-settings" app
+    And "Alice" navigates to the project spaces management page
+    When "Alice" updates the space "team.a" name to "brian team" using the context-menu
+    And "Alice" updates the space "team.b" name to "carol team" using the context-menu
+    And "Alice" updates the space "team.a" quota to "10" using the context-menu
+    And "Alice" updates the space "team.b" quota to "50" using the context-menu
+    And "Alice" disables the space "team.a" using the context-menu
+    And "Alice" disables the space "team.b" using the context-menu
+    And "Alice" enables the space "team.a" using the context-menu
+    And "Alice" enables the space "team.b" using the context-menu
+    And "Alice" disables the space "team.a" using the context-menu
+    And "Alice" disables the space "team.b" using the context-menu
+    And "Alice" deletes the space "team.a" using the context-menu
+    And "Alice" deletes the space "team.b" using the context-menu
+    Then "Alice" should not see the following spaces
+      | id     |
+      | team.a |
+      | team.b |
+    And "Alice" logs out
+    And "Brian" logs out
+    And "Carol" logs out
```

Comment:
if  no action left for user Carol and Brian maybe we could close them above where their action finishes